internal/pkg/deploy/cloudformation/stack/lb_grpc_web_service_integration_test.go

```diff
@@ -103,7 +103,7 @@ func TestGrpcLoadBalancedWebService_Template(t *testing.T) {
 			expectedBytes := []byte(expected)
 			mExpected := make(map[interface{}]interface{})
 			require.NoError(t, yaml.Unmarshal(expectedBytes, mExpected))
-			require.Equal(t, mExpected, mActual)
+			compareStackTemplate(t, mExpected, mActual)
```

Comment:
Seems like `mActual` should go before `mExpected`
```suggestion
			compareStackTemplate(t, mActual, mExpected)
```

Reply:
I wonder why this is not complaining about `Metadata` being absent - `mExepcted` doesn't have `Metadata` it seems

Reply:
https://github.com/aws/copilot-cli/pull/4155/files#diff-c34d543bad6d3e1ac6437fd95329f5694bd27e677804b1074f8371ec3c405933R447 i adjusted the order to be align with the golang test expect.